### PR TITLE
remove .git suffix from links

### PR DESCRIPTION
**Pull request by obfusk**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -94,6 +94,7 @@ class App:
     def __init__(self, name: str, repo: str):
         self.name = name
         self.repo = repo
+        self.link = repo[:-4] if repo.endswith(".git") else repo
```
Comment (TheLastProject) [inline main.py]: Python 3.9 offers a cleaner way: https://docs.python.org/3.9/library/stdtypes.html?highlight=removeprefix#str.removesuffix

Comment (obfusk) [reply main.py]: Yeah. Wasn't sure which version you were using and whether you cared about compatibility with lower versions. Also, you're not using `removesuffix()` elsewhere. But I can use it if you prefer. Want me to use it in the other code parts removing a prefix/suffix too?

Comment (TheLastProject) [reply main.py]: We can update it anywhere I guess.

Don't really care about backward compatibility given it's a very fire-and-forget script.

Comment (obfusk) [reply main.py]: > Don't really care about backward compatibility given it's a very fire-and-forget script.

Fair. I'm used to supporting >= 3.8 for my Python tools.

And done.

**Pull request merged**